views/js/qtiCreator/widgets/choices/inlineChoice/states/Question.js

```diff
@@ -72,7 +72,8 @@ define([
                 toolbar: [
                     {
                         name: 'insert',
-                        items: ['SpecialChar']
+                        items: ['SpecialChar'],
+                        mathJax: false
```

Comment:
toolbar contains only sets of plugins
options like `qtiMedia` is outside
please move `mathJax` near other options